components/library/webm/libvpx/Makefile

```diff
@@ -0,0 +1,62 @@
+#
```

Comment:
Moving them to specific directory under category should not be desired. With fixed structure, we know the maximum depth where components can be found. It also copies behaviour of FreeBSD ports or pkgsrc.